### DISCUSSION TITLE
Welcome to Hextor Discussions!

### DISCUSSION THREAD

**Discussion by digitalw0lf** [digitalw0lf/hextor#1] [General]
Any questions, bug reports, suggestions and feedback on Hextor are welcome here.

Also feel free to reach me on info@hextor.net

**Comment by leon9078** [comment]
Hi @digitalw0lf 
First of all, I'd like to thank you for this amazing software! I had been using it for a while and, compared to other open source solutions I found for Windows, Hextor is very powerful especially with Find/Replace with subexpressions.
Since I want to try your latest fixes/improvements, I tried building the software from source, but I've end up having issues..

You also recommend this in your readme

> ### Building from sources
> 
> I recommend using pre-built binaries from Releases page.

Can you please make a new Release of Hextor (v0.15)?
Has been a while since your last release (end of 2022), and already been several commits
I'm looking forward to try the new "Search with unknown half of byte (e.g. '?1')" feature.

Thanks in advance!

**Reply by digitalw0lf** [reply]
Hi @leon9078 . Thanks for the feedback!
Yes, it's been a while since I last touched Hextor.
So, here you are: https://github.com/digitalw0lf/hextor/releases/tag/v0.15
